Enhance: EndlessOS (#50)
Malix-off/awesome_atomic: readme.md

  - [awesome-nix](https://github.com/nix-community/awesome-nix) - A curated list of the best resources in the Nix community
  - Note: The nix community has both an OS and a packaging system and is already large and vibrant, I can't do it justice so just use the list above to dive in
- [Guix System](https://guix.gnu.org/) - a Linux distribution based on the Guix package manager based on Nix

> - [EndlessOS](https://endlessos.com/home/) - A Debian derivative distribution with a read-only root filesystem managed by OSTree, and is Flatpak-only for application delivery and update. One of the most noob-friendly linux distribution

Author: ```suggestion
- [EndlessOS](https://endlessos.com/home/) - A Debian derivative distribution with a read-only root filesystem managed by OSTree, and is Flatpak-only for application delivery and update. One of the easiest (but constraining) linux distribution
```

Reviewer: To be honest I like that even less.

Author: What's the problem here ?
